styles/vikunja/catppuccin.user.css

```diff
@@ -0,0 +1,282 @@
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/vikunja/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Avikunja
+@description Soothing pastel theme for Vikunja
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire*", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+@-moz-document domain('vikunja.guaxinim.online') {
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+    
+    --site-background: @base;
+    --content-heading-color: @text;
+    --card-header-color: @text;
+    --card-color: @text;
+    --body-color: @text;
+    --label-color: @text;
+    --strong-color: @text;
+    --primary: @accent-color;
+    --primary-invert: @base;
+    --switch-view-color: @base;
+    --input-background-color: @base;
+    --input-icon-color: @text;
+    --table-row-hover-background-color: @mantle;
+    --table-head-cell-color: @text;
+    --input-border-color: @overlay0;
+    --link-hover: @accent-color;
+    --white: @base;
+    --grey: @overlay0;
+    --card-content-background-color: @base;
+    --success: @green;
+    --danger: @red;
+    --warning: @yellow;
```

Comment:
On a note, they use `--danger-h` and so on for hsl and some others, you could save yourself some time themeing buttons etc with this

```less
#hslbreakdown(@color, @item) {
        --@{item}-h: hue(@color) !important;
        --@{item}-s: saturation(@color) !important;
        --@{item}-l: lightness(@color) !important;
        --@{item}-light-l: lightness(@color) !important;
        --@{item}-dark-l: lightness(@color) !important;
        --@{item}-invert-l: lightness(@color) !important;
}
    
// then calling them
#hslbreakdown(@peach, warning);
#hslbreakdown(@green, success);
#hslbreakdown(@red, danger);
#hslbreakdown(@accent-color, primary);
```